earth_enterprise/rpms/opengee-fusion/post-uninstall.sh

```diff
@@ -16,14 +16,15 @@
 
 remove_users_and_groups()
 {
-    if getent passwd "$GEFUSIONUSER" > /dev/null; then
-        userdel "$GEFUSIONUSER"
-    fi
+    echo "opengee user $GEFUSIONUSER may now be safely removed."
```

Comment:
Style: 'opengee' -> 'OpenGEE'

Minor: The user can be removed safely only if the system administrator can confirm there are no files owned by that user that can appear on the machine.